src/confd/src/ietf-interfaces.c

```diff
@@ -606,6 +613,10 @@ static sr_error_t netdag_gen_iface(sr_session_ctx_t *session, struct dagger *net
 	if (err)
 		goto err_close_ip;
 
+	err = lag_gen_ports(net, dif, cif, ip);
+	if (err)
+		goto err_close_ip;
+
```

Comment:
Should we move this, along with `bridge_port_gen()`, to a `netdag_gen_iface_lower()` or something?